source/extensions/common/tap/admin.cc

```diff
@@ -85,10 +85,10 @@ Http::Code AdminHandler::handler(absl::string_view, Http::HeaderMap&, Buffer::In
   return Http::Code::OK;
 }
 
-Http::Code AdminHandler::badRequest(Buffer::Instance& response, absl::string_view error) {
-  ENVOY_LOG(debug, "handler bad request: {}", error);
-  response.add(error);
-  return Http::Code::BadRequest;
+Http::Code AdminHandler::badRequest(Buffer::Chunker& response, absl::string_view error) {
+  absl::string_view errorString = absl::string_view{"handler bad request: "} + error;
```

Comment:
PTAL at https://github.com/envoyproxy/envoy/blob/main/STYLE.md for variable naming conventions, etc.

I'd use std::string rather than absl::string_view in this case as the variable lifetime of what you wrote makes me uncomfortable. I'd also use explicit absl::StrCat
```
absl::string_view error_string = absl::StrCat("handler bad request: ", error);
```

Reply:
Got it.